### PR TITLE
Mitchell Standley

### PR DESCRIPTION
**Pull request by Mstandley1985**

### CODE REVIEW

File: lib/venue.rb [hunk 1]
```diff
@@ -1,2 +1,42 @@
...
+  # ...that code wasn't passing. I wanted to get the test to
+  # at least pass, so I went with the following. 
+
+  def over_capacity?
+    if @patrons.include?("Cat")
+      true
+    else
+      false
+    end
+  end
```
Comment (memcmahon) [inline lib/venue.rb:41]: What you were first working with was more on the right track.  You want to compare the number of patrons to the capacity of the venue:

```ruby
def over_capacity?
  @patrons.length > @capacity
end
```

What you ended up with is passing your tests... but does not really model the behavior that we are looking for - not all venues will have a patron 'Cat'

File: lib/venue.rb [hunk 2]
```diff
@@ -1,2 +1,42 @@
...
+  def capacity
+    @capacity
+  end
+
+  def add_patron(patron)
+    @patrons << patron
+  end
+
+  def yell_at_patrons
+    @patrons.map(&:upcase)
```
Comment (memcmahon) [inline lib/venue.rb:19]: This works, but... I'm a bit worried that you used this syntax (which is shorthand for using map with a block) but couldn't get the next method working.  It indicates to me there is some gap in your understanding of either iterative methods (specifically using blocks) or branching logic (modeled in the over_capacity? method).

I would be curious to see if you can make this method work using `each`?

**Review by memcmahon — COMMENTED** [review]
Good start - take a look at my notes and let me know if you have any questions!

Score: 2

**Comment by Mstandley1985** [comment]
Megan,

Thank you for the feedback! I was able to figure out the:

def over_capacity?
  @patrons.length > @capacity
end

shortly after my pull request by looking through a few more of my notes and tinkering with it a bit more. I certainly have a better understanding after doing so. That also would have better led me into an attempt at iteration 4. I just need to continue reinforcing what we’re learning.  I’ll stay on it! 

Thanks again!

Mitchell Standley

**Pull request closed without merging**